pages/portfolio.html

```diff
@@ -1,12 +1,98 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Amy Nguyen</title>
+  <link href="../styles/style.css" rel="stylesheet">
+  <link href="../styles/portfolio.css" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.15.1/devicon.min.css">
 </head>
+
 <body>
-  
+  <header>
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="resume.html">Resume</a>
+      <a href="portfolio.html">My portfolio</a>
+    </nav>
+  </header>
+  <main>
+      <section class="main-content">
+        <div class="portfolio-cards">
+          <div class="portfolio-card">
+            <div class="porfolio-title">
+              <h3>Otters Matching Cards</h3>
+            </div>
+            <div class="portfolio-flip-card">
+              <div class="flip-card-front">
+                <img src="../assets/otters-memory-gameplay.gif" alt="memory-matching-game">
+              </div>
+              <div class="flip-card-back">
+                <div class="portfolio-description">A simple memory matching card game built with HTML, CSS and vanilla JavaScript.</div>
+              </div>
+            </div>
+            <div class="portfolio-static-card">
+              <div class="buttons">
+                <a class="button" href="https://github.com/thaimynguyen/Otters_memory_game">GitHub</a>
+              </div>
+            </div>
+          </div>
+          <div class="portfolio-card">
+            <div class="porfolio-title">
+              <h3>Space Invaders Pygame</h3>
```

Comment:
You left off your `h1` for this page (it should say something like "Portfolio") but even then this skips the `h2` tag.  It's important to make sure your tags could be used to build a table of contents for the page.

See here for more details: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements